test/e2e/serial/tests/configuration.go

```diff
@@ -381,7 +382,102 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 				}
 				return true, nil
 			}).WithTimeout(5*time.Minute).WithPolling(10*time.Second).Should(BeTrue(), "failed to update RTE container with LogLevel %q", operatorv1.Trace)
+		})
+
+		It("[replaces-47674][images][tier2] should be able to modify the configurable values under the NUMAResourcesOperator CR", Label("images", "tier2"), func(ctx context.Context) {
```

Comment:
Although #1170 has not been merged, we can drop the obsolete tags.

Reply:
can you specify what tags are obsolete here please?

Reply:
`[images][tier2]`